setup.py

```diff
@@ -43,6 +44,8 @@ def _read_readme():
         "Programming Language :: Python :: 3.7",
         "Operating System :: OS Independent",
     ],
+    # Avoid bug in Protocol in python 3.9.7
```

Comment:
I'd just add this comment for clarity, WDYT?
```suggestion
    # Avoid bug in Protocol in python 3.9.7. Note that higher and lower versions of Python are fine.
```